README.md

```diff
@@ -68,10 +68,10 @@ To load SchedViz, go to http://localhost:7402/collections
     want to collect a trace from:
 
     ```bash
-    sudo ./trace.sh -out 'Path to directory to save trace in' \
-                    -capture_seconds 'Number of seconds to record a trace' \
-                    [-buffer_size 'Size of the trace buffer in KB'] \
-                    [-copy_timeout 'Time to wait for copying to finish']
+    sudo ./util/trace.sh -out 'Path to directory to save trace in' \
```

Comment:
Could you put this change in its own PR?

Reply:
Done.